.github/workflows/benchmark-pr-http-comment.yml

```
@@ -0,0 +1,48 @@
+        # the job — the script below degrades gracefully when files are absent.
+        continue-on-error: true
+        uses: actions/download-artifact@v4
+        with:
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+          pattern: benchmark-http-pr-*
+          merge-multiple: true
+          path: benchmark-artifact
+
+      - name: Post or update PR comment
+        uses: actions/github-script@v7
```

actionlint (1.7.12)

[error] missing input "script" which is required by action "actions/github-script@v7". all required inputs are "script"

(action)